src/builder.rs

```diff
@@ -298,4 +298,21 @@ macro_rules! impl_api_ep {
         }
         }}
     };
+    (
+        Reize $it:ident: $base:ident -> $resp:ident $ep:expr, $ret:expr $(, $extra:expr)*
```

Comment:
I'd suggest implementing it straight in `src/api/container.rs` as this macros are mostly for methods that appear on multiple structs like `inspect` or `list`. As the time passes I'm personally leaning more towards implementing it directly for each struct rather than by macro as it improves code search-ability.